media-video/ffmpeg/ffmpeg6-6.0.recipe

```diff
@@ -0,0 +1,346 @@
+	devel:libmp3lame$secondaryArchSuffix
+	devel:libogg$secondaryArchSuffix
+	devel:libopus$secondaryArchSuffix
+	devel:libsdl2$secondaryArchSuffix
+	devel:libspeex$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	devel:libtheora$secondaryArchSuffix
+	devel:libvorbis$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+if [ "$effectiveTargetArchitecture" != x86_gcc2 ]; then
+	BUILD_REQUIRES+="
+		devel:libass$secondaryArchSuffix
+		devel:libdav1d$secondaryArchSuffix
+		devel:libfdk_aac$secondaryArchSuffix
+		devel:libfontconfig$secondaryArchSuffix
+		devel:libfreetype$secondaryArchSuffix
+		devel:libfribidi$secondaryArchSuffix
+		devel:libgme$secondaryArchSuffix
+		devel:libgmp$secondaryArchSuffix
+		devel:libopenjp2$secondaryArchSuffix
+		devel:libopenmpt$secondaryArchSuffix
+		devel:libsnappy$secondaryArchSuffix
+		devel:libsoxr$secondaryArchSuffix
+		devel:libvpx$secondaryArchSuffix
+		devel:libwebp$secondaryArchSuffix
+#		devel:libx264$secondaryArchSuffix
+#		devel:libx265$secondaryArchSuffix
+		devel:libxml2$secondaryArchSuffix
+		devel:liblzma$secondaryArchSuffix
+		"
+fi
+BUILD_PREREQUIRES="
+	cmd:awk
+	cmd:cmp
+	cmd:gcc$gccSuffix >= 7
+	cmd:g++$secondaryArchSuffix
+	cmd:ld$gccSuffix
+	cmd:grep
+	cmd:make
+	cmd:perl
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:sed
+	cmd:texi2html
+	cmd:yasm
+	"
+
+PATCH()
+{
+	# patch hard-coded paths to perl
+	sed -i "s,/usr/bin/perl,$portPackageLinksDir/cmd~perl/bin/perl," \
+		doc/Doxyfile \
+		doc/texi2pod.pl
+	# force no-undefined-symbols
+	if [ "$effectiveTargetArchitecture" = x86_gcc2 ]; then
+		sed -i "s/SHFLAGS='-shared/SHFLAGS='-shared -Wl,--no-undefined/g" \
+			configure
+	fi
+}
+
+defineDebugInfoPackage ffmpeg6$secondaryArchSuffix \
+	$libDir/libavcodec.so.$libavcodecVersion \
+	$libDir/libavfilter.so.$libavfilterVersion \
+	$libDir/libavformat.so.$libavformatVersion \
+	$libDir/libavutil.so.$libavutilVersion \
+	$libDir/libswresample.so.$libswresampleVersion \
+	$libDir/libswscale.so.$libswscaleVersion \
+	"$(getPackagePrefix avdevice)/$relativeLibDir"/libavdevice.so.$libavdeviceVersion \
+	"$(getPackagePrefix tools)/$relativeBinDir"/ffmpeg \
+	"$(getPackagePrefix tools)/$relativeBinDir"/ffplay \
+	"$(getPackagePrefix tools)/$relativeBinDir"/ffprobe
+
+BUILD()
+{
+	ccArgs=""
+	extraArgs=""
+	if [ "$effectiveTargetArchitecture" = x86_gcc2 ]; then
+		cc="gcc${gccSuffix/_/-}"
+		ccArgs="--cc=$cc --cxx=g++${secondaryArchSuffix/_/-}
+			--ld=gcc${secondaryArchSuffix/_/-}
+			--host-ld=gcc${secondaryArchSuffix/_/-}"
+
+		# Hack up base headers to make them think we're on GCC2 ABI
+		mkdir -p include_hacks include_hacks/os include_hacks/config
+		pushd include_hacks
+			rm -f os/BeBuild.h config/HaikuConfig.h
+			cp /system/develop/headers/os/BeBuild.h os
+			cp /system/develop/headers/config/HaikuConfig.h config
+			sed -i 's/__GNUC__ == 2/1/g' os/BeBuild.h config/HaikuConfig.h
+		popd
+
+		# Compile gcc_runtime.c (subset of libgcc that GCC 7 utilizes here)
```

Comment:
This comment should refer to GCC 13 now (or generalize it to "modern GCC" or something).

Reply:
Both adjusted, thanks!